src/DataStorage.jl

```diff
@@ -0,0 +1,109 @@
+module DataStorage
+
+## Imports
+import Base: size #must import to add a definition to size
+
+##Exports
+export DataContainer, PairedDataContainer
+export size
+export get_data, get_inputs, get_outputs
+
+## Objects
+"""
+    struct DataContainer
+
+struct to store data samples as columns in an array
+"""
+struct DataContainer{FT <: Real}
+    #stored data, each piece of data is a column [number of samples x data size]
```

Comment:
Wouldn't `stored_data` be an array `[number of samples x data size]`, where number of samples is the size of the ensemble and data size is the number of parameters?

Reply:
should be [data_dim x n_samples]. Changed!